packages/compass-workspaces/src/types.ts

```diff
@@ -18,6 +20,7 @@ export type CollectionsWorkspace = {
 export type CollectionWorkspace = {
   type: 'Collection';
   namespace: string;
+  subTab: CollectionSubtab;
```

Comment:
Can we type this as string union instead of a enum? Makes a bit easier to integrate and means we don't need to re-export this type from compass-web so that mms can use it (and easier for us to use cross plugins)

Reply:
yeah no strong preference here. will make the change